packages/itwin/property-grid/package.json

```
@@ -34,7 +34,7 @@
     "copy:esm": "cpx \"./src/**/*.scss\" ./lib/esm",
     "cover": "nyc npm run test",
     "extract-api": "betools extract-api --entry=property-grid-react",
-    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
+    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2 && npx stylelint \"**/*.scss\"",
```

`npx` shouldn't be needed.
```suggestion
    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2 && stylelint \"./src/**/*.scss\"",
```

Maybe separate scripts for `eslint` and `stylelint`?
```
"lint": "npm run lint:eslint && npm run lint:stylelint",
"lint:eslint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
"lint:stylelint": "stylelint \"./src/**/*.scss\"",
```